eng/tools/tasks/Microsoft.DotNet.UnifiedBuild.Tasks/Microsoft.DotNet.UnifiedBuild.Tasks.csproj

```diff
@@ -6,11 +6,11 @@
 
   <ItemGroup>
     <!-- IncludeAssets=compile to treat these packages as targeting-packs only. The assemblies are available in the SDK. -->
-    <PackageReference Include="Microsoft.Build.Tasks.Core" IncludeAssets="compile" />
-    <PackageReference Include="Microsoft.Build.Utilities.Core" IncludeAssets="compile" />
-    <PackageReference Include="NuGet.Protocol" IncludeAssets="compile" />
-    <PackageReference Include="NuGet.ProjectModel" IncludeAssets="compile" />
-    <PackageReference Include="Newtonsoft.Json" IncludeAssets="compile" />
+    <PackageReference Include="Microsoft.Build.Tasks.Core" IncludeAssets="compile" NoWarn="NU1901;NU1902;NU1903;NU1904" />
```

Comment:
Do these in-the-vmr projects actually need to reference Tasks.Core? Usually you don't and I think I got it building locally without (but I rebooted and lost my context there, sorry).

Reply:
Good think to look at in a follow-up

Reply:
Yeah absolutely not blocking!